apps/consumer-client/package.json

```diff
@@ -43,13 +46,15 @@
     "@babel/core": "^7.0.0",
     "@esbuild-plugins/node-globals-polyfill": "^0.2.3",
     "@esbuild-plugins/node-modules-polyfill": "^0.2.2",
+    "@pcd/artifacts": "0.0.1",
     "@pcd/eslint-config-custom": "*",
     "@pcd/tsconfig": "*",
     "@types/react": "^18.2.6",
     "@types/react-dom": "^18.0.7",
     "@types/styled-components": "^5.1.26",
     "@typescript-eslint/eslint-plugin": "^5.56.0",
     "@typescript-eslint/parser": "^5.56.0",
+    "download": "^8.0.0",
```

Comment:
are you sure this dependency needs to be added?

Reply:
Good catch, we don't need it anymore